### PR TITLE
Code cleanup

### PR DESCRIPTION
**Pull request by KrishnaSai2020**
I am back again after a long time with a whole host of code simplifying from simplifying chain expressions to removing redundant comments to incredibly annoying spelling mistakes.

### CODE REVIEW

**Comment by KrishnaSai2020** [comment]
@pnhofmann

**Review by FatalTrip — APPROVED** [review]

**Comment by KrishnaSai2020** [comment]
Could I geet an update on this PR please?

File: jarviscli/plugins/caesar_cipher.py [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 
 @plugin("caesar cipher")
-def caesar_cipher_converter(jarvis, str):
+def caesar_cipher_converter(jarvis):
```
Comment (pnhofmann) [inline jarviscli/plugins/caesar_cipher.py:7]:
```suggestion
def caesar_cipher_converter(jarvis, s):
```

Second parameter is required (but you are right, naming a parameter str is not good).

File: jarviscli/plugins/caesar_cipher.py [hunk 2]
```diff
@@ -52,7 +52,7 @@ def plain_to_cipher(jarvis):
                 converted += i
         else:
             x = ord(i)
-            if x >= 192 and x <= 255:
+            if 192 <= x <= 255:
```
Comment (pnhofmann) [inline jarviscli/plugins/caesar_cipher.py:55]: Honestly didn't knew, this works in Python. But this is really nice! Thanks ;).

**Comment by pnhofmann** [comment]
Thanks! Good work, you really found lots of stuff!

File: jarviscli/plugins/caesar_cipher.py (hunk 2 above)
Comment (KrishnaSai2020) [reply jarviscli/plugins/caesar_cipher.py:55]: Np!

Comment (KrishnaSai2020) [reply jarviscli/plugins/caesar_cipher.py:55]: The way I think about it is if it works in English it should work in python.

**Pull request merged**